content/docs/0.19.x/reference/files/values/index.md

```diff
@@ -27,10 +27,16 @@ so can be used to temporarily override a value when necessary.
 
 ## Files
 
-* Default [values.yaml](https://github.com/keptn/keptn/blob/master/helm-service/chart/values.yaml#L10) file: 
+* Default [values.yaml](https://github.com/keptn-contrib/helm-service/blob/main/chart/values.yaml) file 
 
 ## Differences between versions
 
+**Keptn 0.19.0 changes**
+
+The *helm-service* integration is moved out of the keptn/keptn github repository
+and into the keptn-contrib repository in Release 0.19.x
+and so the path to the *values.yaml* file is changed.
```

Comment:
JMeter service should also be added here, since the same thing happened to it.

Reply:
Done.  I was thinking that the values.yaml file was only affected by the new path for helm-service but I suppose it isn't a bad idea to make this statement as much as possible.